_posts/2023-12-01-improving-document-retrieval-with-spade-semantic-encoders.md

```diff
@@ -0,0 +1,446 @@
+---
+layout: post
+title:  Improving document retrieval with sparse semantic encoders
+authors:
+  - zhichaog
+  - xinyual
+  - dagney
+  - yych
+date: 2023-12-01 01:00:00 -0700
+categories:
+    - technical-posts
+meta_keywords: Improve search relevance, semantic search in OpenSearch 2.11, search with sparse encoders, neural search
+meta_description: Improve search relevance with OpenSearch 2.11 when you build your high relevance & high speed search engine using neural sparse search.
```

Comment:
Please update the meta keywords and description with the following: 

**meta_keywords:** search relevance, neural sparse search, semantic search, semantic search with sparse encoders 
**meta_description:** Learn how the neural sparse framework in OpenSearch 2.11 can help you improve search relevance and optimize semantic searches with spare encoders using just a few APIs.